src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.html

```diff
@@ -55,9 +54,16 @@
       </div>
     </div>
 
+    <p *ngIf="invalidAttemptsCount > 6 && validAttemptsCount > 0"
+       i18n><b>You have entered invalid credentials {{ invalidAttemptsCount }} times consecutively since your last successful login. You have {{ validAttemptsCount }} attempts left.</b></p>
```

Comment:
I think simplifying this error would be better. WDYT? Also changed the color to `vv.$danger` and aligned it to right.
![changes](https://user-images.githubusercontent.com/71764184/97849220-40e37080-1d18-11eb-8c3a-7d413d739f93.png)

Reply:
From a security POV, I'm not sure if we should give a potential hacker any hints about how many attempts they have to guess the correct login credentials. The error message/code they see should always be the same and it should not even be possible to determine from the error message if it's the user name or password that was entered incorrectly.

Reply:
Yeah, I agree to that. So then as a warning to user, we could show a message for the last try or maybe for the last 2 tries like `Invalid Credentials. 1 Attempt Remaining`, or use the same message in this PR. (Although I still think the message is a bit long). @LenzGr ?